src/networking.c

```diff
@@ -3438,6 +3438,28 @@ sds getAllClientsInfoString(int type, int hide_user_data) {
     return o;
 }
 
+sds getAllFilteredClientsInfoString(clientFilter *client_filter, int hide_user_data) {
+    listNode *ln;
+    listIter li;
+    client *client;
+    sds o = sdsnewlen(SDS_NOINIT, 200 * listLength(server.clients));
```

Comment:
The number of connected clients can be thousands, so maybe we shouldn't allocate this huge number. I suggest we initialize it to a fixed 500 bytes or so.

Reply:
finally done :D